diff --git a/src/device.c b/src/device.c
index 3ebda66..ff10044 100644
--- a/src/device.c
+++ b/src/device.c
@@ -435,6 +435,8 @@ struct device_event *device_read_event(struct device *dev)
 				ev.code = KEYD_ZOOM;
 			else if (ev.code >= BTN_DIGI
 				 && ev.code <= BTN_TOOL_QUADTAP);
+			else if (ev.code == 316); // 13c
+			else if (ev.code == 317); // 13d
 			else {
 				keyd_log("r{ERROR:} unsupported evdev code: 0x%x\n", ev.code);
 				return NULL;
